nodezator/fontsman/cache.py

```diff
@@ -45,8 +45,8 @@ def __missing__(self, key):
         """
         ### we create a font map for the key, store and
         ### return it
-        font_map = self[key] = FontsMap(key)
-
+        font_map = FontsMap(key)
+        self[key] = font_map
```

Comment:
note: for clarity's sake

The original issue, below, is one where this `self[key]=bla` is missing. In my experience, the code-smell from multiple statements on one line can encourage situations of overlooking logic by a reader. I believe that's what has happened here.